pyproject.toml

```diff
@@ -20,6 +20,7 @@ dependencies = [
     "fastapi[standard]>=0.112.1",
     "psutil>=6.0.0",
     "tomlkit>=0.13.2",
+    "pdm-backend>=2.3.3",
```

Comment:
Did you mean to move this to a full dependency?

Reply:
no, that was an accident. Will fix